index.html

```diff
@@ -3346,9 +3346,20 @@ <h3>Vertical text</h3>
 
 
 <section id="typ_bidi" class="subtopic">
-<h3><em>RTL/bidi text (TBD)</em></h3>
+<h3><em>RTL/bidi text</em></h3>
 
 <p class="reviewComments"><a href="https://github.com/w3c/i18n-activity/labels/t%3Atyp_bidi_x" target="_blank">See related review comments.</a></p>
+<aside class="links" id="links_cursive">
+<p class="links_title">Useful background and overviews for this section</p>
+<ul>
+<li class="w3"><p class="link"><a href="https://www.w3.org/TR/typography/#bidi_text">Bidirectional text</a>, in the Language enablement index.</p></li>
+</ul>
+</aside>
+
+<div class="req" id="italics_left">
+	<p class="advisement">Specifications enabling italicization of text should support sloping the letters to the left.</p>
```

Comment:
probably this should say "obliquing" instead of "italicization". I would suggest making this more generic as well. Perhaps:

```suggestion
<p class="advisement">Specifications enabling the italicization or obliquing of text SHOULD provide for characters to slope either to the right or to the left according to the needs of the specific language.</p>
```

Reply:
Or you could just say:
Specifications that enable sloping of letterforms SHOULD...

Reply:
Thanks. I have updated the PR.